packages/splitter/splitter.stories.tsx

```diff
@@ -7,16 +7,16 @@
 
 import React, { useState } from 'react';
 import { SplitterContainer, IUseSplitterReturnValue, useSplitter } from './src';
-import { SplitterOrientation, SplitterType } from './src/useSplitter';
+import { SplitterOrientation, SplitterType, SplitterPosition } from './src/useSplitter';
```

Comment:
Should these exports be added to `index.ts` for reuse? I'm not sure this is totally consistent with other container packages or not, but the imports are shown in the README.

Reply:
Fixed in https://github.com/zendeskgarden/react-containers/pull/405/commits/72feeb0d9c03ec822bb561b57da6c3adc01a7996